docs/postmortems/0000-00-00-template.md

```diff
@@ -0,0 +1,67 @@
+---
+date: YYYY-MM-DD
+authors: spirit-developer
+---
+
+# Title of Postmortem
+
+**Summary**
+
+<!-- What is a one or two-line summary of the event that occurred? -->
+
+**Impact**
+
+<!-- What was the scope of impact from the event occurring? How many teams were
+impacted? -->
+
+**Root causes**:
+
+<!-- Looking back, what ended up being the main reasons why this event occurred?
+-->
+
+**Detection**
+
+<!-- How did we find out or discover that this event had occurred? -->
+
+**Resolution**
+
+<!-- How did we end up addressing this event to mitigate the impact? -->
+
+**Action Items**
+
+<!-- What are the action items that came out of this postmortem? Reference
+issues and Pull Requests in the "Bug" column with the appropriate owners -->
+
+| Action item | Owner | Issue |
+| ----------- | ----- | ----- |
+
+## Lessons Learned
+
+**What went well**
+
+- Item A
+- Item B
+- Item C
+
+**What went wrong**
+
+- Item A
+- Item B
+- Item C
+
+**Where we got lucky**
+
+- Item A
+- Item B
+- Item C
+
+## Timeline
+
+2024-01-01 **(all times in CET/CEST)**
```

Comment:
maybe we should place the time information in a separate line or in the title? You can see that in the filled template it looks weird as it feels like it applies only to the first date.

Reply:
Fixed.